packaging/nnstreamer.spec

```diff
@@ -885,6 +904,13 @@ NNStreamer's datareposrc/sink plugins for reading and writing files in MLOps Dat
 %define enable_onnxruntime -Donnxruntime-support=disabled
 %endif
 
+# Support executorch
+%if 0%{?executorch_support}
+%define enable_executorch -Dexecutorch-support=enabled
+%else
+%define enable_executorch -Dexecutorch-support=disabled
```

Comment:
Make executorch-llama-support explicit or add a comment that executorch-llama-support will be enabled alongside.